packages/bodiless-cli/src/helpers/getPackageTailwindConfig.ts

```diff
@@ -0,0 +1,45 @@
+/**
```

Comment:
SInce w eno longer need a generate-tailwind-configs command, maybe let's move this (and mergeConfigs) to fclasses?